src/common/ApiResponse/createApiResponse.ts

```diff
@@ -1,21 +1,41 @@
+import { v4 as uuid } from "uuid";
 import { AxiosResponse } from "axios";
 
 import { ApiResponse } from "./types";
 import { isErrorStatusCode } from "./utils";
 
+export type CreateApiResponseOptions = { message: string; id?: string };
```

Comment:
Added the optional `id` prop to allow for easier testing via dependency injection.

I thought about mocking the uuid package, but I prefer not to mock my Application as much as possible.
 
I don't feel to great about poking a hole in my Application's reality, but I think this is the lesser of two evils.